### PR TITLE
chore: Move some examples to test/produces-output

### PR DESCRIPTION
**Pull request by eriksvedang**
Will make it easier for people to find the interesting bits in the 'examples' directory.

### CODE REVIEW

**Comment by TimDeve** [comment]
You've got a few .DS_Store files in there.

**Comment by eriksvedang** [comment]
@TimDeve Oh, dang! Thanks 👍

File: scripts/run_carp_tests.sh [hunk 1]
```diff
@@ -43,7 +43,6 @@ echo "Compile-only examples"
 compileOnlyExamples="\
                    examples/mutual_recursion.carp \
                    examples/guessing_game.carp \
-                   examples/unicode.carp \
```
Comment (jacereda) [inline scripts/run_carp_tests.sh]: We should restore this one if #994 is applied.

Comment (eriksvedang) [reply scripts/run_carp_tests.sh]: Awesome, I'll put it back!

**Comment by eriksvedang** [comment]
Also closes #991.

**Comment by eriksvedang** [comment]
Oh, snap – we don't have `clock_gettime` on Windows so the benchmarks won't build. Do we have a workaround for that already?

**Comment by TimDeve** [comment]
Could steal a replacement implementation from stack overflow...

**Comment by jacereda** [comment]
> Oh, snap – we don't have `clock_gettime` on Windows so the benchmarks won't build. Do we have a workaround for that already?

I think I want to take a look at that after this gets merged. IMHO `System_nanotime` should return integer nanoseconds. Keep in mind things like the GBA don't have FP hardware and any FP operation there is extremely expensive.

**Comment by eriksvedang** [comment]
> > Oh, snap – we don't have `clock_gettime` on Windows so the benchmarks won't build. Do we have a workaround for that already?
> 
> 
> 
> I think I want to take a look at that after this gets merged. IMHO `System_nanotime` should return integer nanoseconds. Keep in mind things like the GBA don't have FP hardware and any FP operation there is extremely expensive.

Yes please! This was just a quick hack.

**Pull request merged**